.github/workflows/rust.yml

```
@@ -0,0 +1,26 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Format
+      run: cargo fmt --verbose
```

CIなので修正するのではなく修正漏れが無いかの確認がしたい

```suggestion
      run: cargo fmt --verbose -- --check
```